launch/ekf.launch

```diff
@@ -0,0 +1,19 @@
+<launch>
+
+ <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" clear_params="true" output="screen">
+   <param name="frequency" value="20"/>
+
+   <param name="odom_frame" value="odom"/>
```

Comment:
Line 6- 9. Do we want to use remap here?

Reply:
These params are for frames, not topics so remap does not apply